entity-service-impl/src/main/java/org/hypertrace/entity/service/util/DocStoreConverter.java

```diff
@@ -128,17 +128,13 @@ public static Filter getTenantIdEqFilter(String tenantId) {
 
   private static Filter transform(AttributeFilter filter) {
     if (filter.hasAttributeValue()) {
-      if (ATTRIBUTES_LABELS_FIELD_NAME.equals(filter.getName())
```

Comment:
Can we get rid of the `ATTRIBUTES_LABELS_FIELD_NAME` constant from the class?

Reply:
done